webapp/src/main/java/com/box/l10n/mojito/rest/asset/AssetWS.java

```diff
@@ -238,6 +243,24 @@ public XliffExportBody xliffExport(@PathVariable("assetId") long assetId, @Reque
         return new XliffExportBody(normalizedContent);
```

Comment:
I think that function should be refactored to get the content from the the database now.